src/Widgets/PricingWidget/PricingWidgetEditingConfig.js

```diff
@@ -67,4 +67,87 @@ Scrivito.provideEditingConfig("PricingWidget", {
       url: "https://scrivito.com",
     }),
   },
+  validations: [
+    [
+      "currency",
+
+      currency => {
+        if (!currency) {
+          return { message: "The currency must be set.", severity: "error" };
+        }
+      },
+    ],
+    [
+      "smallPlanPrice",
+
+      smallPlanPrice => {
+        if (!smallPlanPrice) {
+          return {
+            message: "The small plan price must be set.",
+            severity: "error",
+          };
+        }
+      },
+    ],
+    [
+      "mediumPlanPrice",
+
+      mediumPlanPrice => {
+        if (!mediumPlanPrice) {
+          return {
+            message: "The medium plan price must be set.",
+            severity: "error",
+          };
+        }
+      },
+    ],
+    [
+      "largePlanPrice",
+
+      largePlanPrice => {
+        if (!largePlanPrice) {
+          return {
+            message: "The large plan price must be set.",
+            severity: "error",
+          };
+        }
+      },
+    ],
+    [
+      "smallPlanButton",
+
+      smallPlanButton => {
+        if (!smallPlanButton) {
+          return {
+            message: "The url for small plan's button must be set.",
+            severity: "error",
+          };
+        }
+      },
+    ],
+    [
+      "mediumPlanButton",
+
+      mediumPlanButton => {
+        if (!mediumPlanButton) {
+          return {
+            message: "The url for medium plan's button must be set.",
+            severity: "error",
```

Comment:
Let us make this consistent with the button widget - so a warning, not an error (with appropriate messaging)